docs/tutorials/implement-pipeline.md

```diff
@@ -0,0 +1,221 @@
+# Implement a pipeline
+
+This tutorial shows how to implement a [Pipeline](/csia-pme/reference/pipeline) in the CSIA-PME project step by step.
+It will guide you through the process of creating a [Pipeline](/csia-pme/reference/pipeline) to detect faces in an image and blur them.
+
+## Tutorial
+
+### Prerequisites
+
+To follow this tutorial, you need to have the following tools installed:
+
+- [Python 3.10](https://www.python.org/downloads/)
+- An IDE (e.g. [Visual Studio Code](https://code.visualstudio.com/))
+
+### Prepare the engine
+
+To implement a [Pipeline](/csia-pme/reference/pipeline), you need to have a running [Engine](/csia-pme/reference/engine). If you don't have one, you can follow the [Engine](/csia-pme/reference/engine) documentation.
+
+!!! optional "Run the Webapp"
+    For a better experience you can run the webapp by following the [webapp reference](/csia-pme/reference/webapp/) documentation.
+
+### Prepare the services
+
+The first step is to prepare the services that will be used in the [Pipeline](/csia-pme/reference/pipeline). In this tutorial, we will use the following services:
+
+- [Face detection](https://face-detection-csia-pme.kube.isc.heia-fr.ch/docs)
+- [Image blur](https://image-blur-csia-pme.kube.isc.heia-fr.ch/docs)
+
+#### Launch the services
+
+If the services are not running you can follow the explanations in the reference documentation to start them.
+
+- [Face detection](/csia-pme/reference/face-detection/)
+- [Image blur](/csia-pme/reference/image-blur/)
+
+### Create the pipeline
+
+The [Pipeline](/csia-pme/reference/pipeline) is created by posting a JSON object to the `/pipelines` endpoint of the [Engine](/csia-pme/reference/engine). Create a file named `face-blur-pipeline.json` in your IDE with the following code:
+
+``` json
+{
+    "name": "Face Blur",
+    "slug": "face-blur",
+    "summary": "Face Blur",
+    "description": "Face Blur",
+    "data_in_fields": [
+        {
+            "name": "image",
+            "type": [
+                "image/jpeg",
+                "image/png"
+            ]
+        }
+    ],
+    "data_out_fields": [
+        {
+            "name": "result",
+            "type": [
+                "image/jpeg",
+                "image/png"
+            ]
+        }
+    ],
+    "steps": [
+        {
+            "identifier": "face-detection",
+            "needs": [],
+            "inputs": ["pipeline.image"],
+            "service_id": "00000000-0000-0000-0000-000000000000" // change this id with the id of your face detection service
```

Comment:
Do you think it's a good idea to highlight these lines so it's easier for the reader to see where to find the service ID?